js/stock-exchange.js

```diff
@@ -0,0 +1,153 @@
+const STOCKS_BASE_URL =
+    "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com";
+const STOCKS_SEARCH_LIMIT = 10;
+const SYMBOL_PREFIX = "$";
+const API_PROFILE_COMPANY_LIMIT = 3;
+
+let search = document.getElementById("btnStockSearch");
+let inputStockSearch = document.getElementById("inputStockSearch");
+var lastSymbol = "";
+
+inputStockSearch.addEventListener("keypress", function (event) {
+    if (event.key === "Enter") {
+        event.preventDefault();
+
+        // Enter key on input works as button click, makes Fibonacci calculation
+        document.getElementById("btnStockSearch").click();
+    }
+});
+
+search.addEventListener("click", function () {
+
+    removeStockProfileInfo();
+
+    let searchText = document.getElementById("inputStockSearch").value;
+
+    if (searchText !== "") {
+        let searchURL = `${STOCKS_BASE_URL}/api/v3/search?query=${searchText}&amp;limit=${STOCKS_SEARCH_LIMIT}&amp;exchange=NASDAQ`;
```

Comment:
your query parameters do not work because your are using "&amp;". replace it with just "&"